### PR TITLE
thank you page content

### LINKED ISSUE #564: Landing page (gated documents) "thank you" content
https://www.moleculardevices.com/en/assets/ebook/br/elisa?page=thankyou
https://www.moleculardevices.com/en/assets/user-guide/dd/cns/axon-guide-to-electrophysiology-and-biophysics-laboratory-techniques?page=thankyou
https://www.moleculardevices.com/en/assets/tutorials-videos/dd/img/ananda-neurite-untangled?page=thankyou

### PR DESCRIPTION
**Pull request by paolomoz**
Fix #564

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/ebook/br/elisa?page=thankyou
- After: https://thank-you-lp--moleculardevices--hlxsites.hlx.page/en/assets/ebook/br/elisa?page=thankyou

### CODE REVIEW

**Review by lydiapuric — CHANGES_REQUESTED** [review]
Mobile version is not styled and is broken
Screenshot 2023-06-06 at 11 49 31

File: scripts/scripts.js [hunk 1]
```diff
@@ -95,19 +95,8 @@ function optimiseHeroBlock(main) {
...
-  if (lastSection.classList.contains('wave')) return;
-  const hasWaveFragment = lastSection.firstElementChild && lastSection.firstElementChild.childElementCount === 1 && lastSection.querySelector('.fragment');
-  if (hasWaveFragment) return;
-
-  const waveSection = document.createElement('div');
-  waveSection.classList.add('section');
-  waveSection.classList.add('wave');
-  waveSection.setAttribute('data-section-status', 'initialized');
-  lastSection.after(waveSection);
+  const skipWave = document.querySelector('.page-tabs, .thankyou, .section.wave:last-of-type, .section:last-of-type div:first-of-type .fragment:only-child');
```
Comment (paolomoz) [inline scripts/scripts.js:98]: @mhaack could you please check if this change is acceptable? 
pros: easier to add special cases + 90% less code
cons: last minute change, maybe you prefer to keep code and make small patches to it. LMK

Comment (mhaack) [reply scripts/scripts.js:98]: Yes, looks ok to me

File: styles/styles.css [hunk 2]
```diff
@@ -1313,3 +1313,13 @@ main .section[aria-hidden="true"] {
...
 }
+
+.thankyou main .section {
+  display: none;
+}
+
+.thankyou main .section.hero-container,
+.thankyou main .section.hero-advanced-container,
+.thankyou main .section.thank-you-page {
+  display: block;
```
Comment (mhaack) [inline styles/styles.css:1324]: Why this in styles.css and not in landing-page.css?

**Comment by paolomoz** [comment]
@mhaack @lydiapuric your suggestions have been address; please review. thanks

**Review by mhaack — APPROVED** [review]

**Pull request merged**